src/melee/ft/chara/ftMasterHand/dir.dox

```diff
@@ -0,0 +1,3 @@
+/** @dir
+ * @defgroup ftmasterhand ftMasterHand
+ */
```

Comment:
Again, don't really get how Doxygen modules work at the moment. I'll figure it out in a future PR.